Packet++/header/HttpLayer.h

```diff
@@ -207,201 +207,217 @@ namespace pcpp
 			/** 405 Method Not Allowed */
-			Http405MethodNotAllowed,
+			Http405MethodNotAllowed = 405,
 			/** 406 Not Acceptable */
-			Http406NotAcceptable,
+			Http406NotAcceptable = 406,
 			/** 407 Proxy Authentication Required */
-			Http407ProxyAuthenticationRequired,
+			Http407ProxyAuthenticationRequired = 407,
 			/** 408 Request Timeout */
-			Http408RequestTimeout,
+			Http408RequestTimeout = 408,
 			/** 409 Conflict */
-			Http409Conflict,
+			Http409Conflict = 409,
 			/** 410 Gone */
-			Http410Gone,
+			Http410Gone = 410,
 			/** 411 Length Required */
-			Http411LengthRequired,
+			Http411LengthRequired = 411,
 			/** 412 Precondition Failed */
-			Http412PreconditionFailed,
+			Http412PreconditionFailed = 412,
 			/** 413 RequestEntity Too Large */
-			Http413RequestEntityTooLarge,
+			Http413RequestEntityTooLarge = 413,
 			/** 414 Request-URI Too Long */
-			Http414RequestURITooLong,
+			Http414RequestURITooLong = 414,
 			/** 415 Unsupported Media Type */
-			Http415UnsupportedMediaType,
+			Http415UnsupportedMediaType = 415,
 			/** 416 Requested Range Not Satisfiable */
-			Http416RequestedRangeNotSatisfiable,
+			Http416RequestedRangeNotSatisfiable = 416,
 			/** 417 Expectation Failed */
-			Http417ExpectationFailed,
-			/** 418 I'm a teapot */
-			Http418ImATeapot,
-			/** 419 Authentication Timeout */
-			Http419AuthenticationTimeout,
-			/** 420 (various messages) */
-			Http420,
+			Http417ExpectationFailed = 417,
+			/** 418 Unused **/
+			/** 419-420 unassigned **/
+			/** 421 Misdirected Request */
+			Http421MisdirectedRequest = 421,
 			/** 422 Unprocessable Entity */
-			Http422UnprocessableEntity,
+			Http422UnprocessableEntity = 422,
 			/** 423 Locked */
-			Http423Locked,
+			Http423Locked = 423,
 			/** 424 Failed Dependency */
-			Http424FailedDependency,
+			Http424FailedDependency = 424,
+			/** 425 Too Early */
+			Http425TooEarly = 425,
 			/** 426 Upgrade Required */
-			Http426UpgradeRequired,
+			Http426UpgradeRequired = 426,
+			/** 427 Unassigned */
 			/** 428 Precondition Required */
-			Http428PreconditionRequired,
+			Http428PreconditionRequired = 428,
 			/** 429 Too Many Requests */
-			Http429TooManyRequests,
+			Http429TooManyRequests = 429,
+			/** 430 Unassigned */
 			/** 431 Request Header Fields Too Large */
-			Http431RequestHeaderFieldsTooLarge,
-			/** 440 Login Timeout */
-			Http440LoginTimeout,
-			/** 444 No Response */
-			Http444NoResponse,
-			/** 449 Retry With */
-			Http449RetryWith,
-			/** 450 Blocked by Windows Parental Controls */
-			Http450BlockedByWindowsParentalControls,
-			/** 451 (various messages) */
-			Http451,
-			/** 494 Request Header Too Large */
-			Http494RequestHeaderTooLarge,
-			/** 495 Cert Error */
-			Http495CertError,
-			/** 496 No Cert */
-			Http496NoCert,
-			/** 497 HTTP to HTTPS */
-			Http497HTTPtoHTTPS,
-			/** 498 Token expired/invalid */
-			Http498TokenExpiredInvalid,
-			/** 499 (various messages) */
-			Http499,
```

Comment:
Why did we remove these values?

Reply:
answered above, let's discussed in the above thread